src/drivers/README.md

```
@@ -48,102 +61,224 @@ await client.$transaction(async (tx) => {
+```ts
+// ✅ New approach - TransactionBoundDriver
+driver._transaction((tx) => {
+  const txDriver = new TransactionBoundDriver(driver, tx);
+  // All operations on txDriver automatically use the transaction
+  await txDriver._execute(query); // Uses tx internally
+});
+```
+
+### How It Works
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)